hedera-node/src/main/java/com/hedera/services/fees/calculation/AwareFcfsUsagePrices.java

```diff
@@ -114,12 +116,12 @@ public FeeData pricesGiven(HederaFunctionality function, Timestamp at) {
 			FeeData usagePrices = functionUsagePrices.get(function);
 			Objects.requireNonNull(usagePrices);
 			return usagePrices;
-		} catch (Exception ignore) {
+		} catch (Exception e) {
 			log.warn(
-					"Only default usage prices available for function {} @ {}.{}!",
+					"Only default usage prices available for function {} @ {}!",
 					function,
-					at.getSeconds(),
-					at.getNanos());
+					Instant.ofEpochSecond(at.getSeconds(), at.getNanos()),
+					e);
```

Comment:
Seems missing one pair of `{}` in `log.warn()`. Should be better if the `log.warn()` is tested.